iroha/iroha.py

```diff
@@ -341,17 +341,27 @@ class IrohaGrpc(object):
     Possible implementation of gRPC transport to Iroha
     """
 
-    def __init__(self, address=None, timeout=None, secure=False):
+    def __init__(self, address=None, timeout=None, secure=False, root_certificates=None,
+                  private_key=None,
+                  certificate_chain=None):
         """
         Create Iroha gRPC client
         :param address: Iroha Torii address with port, example "127.0.0.1:50051"
         :param timeout: timeout for network I/O operations in seconds
         :param secure: enable grpc ssl channel
+        :param root_certificates: The PEM-encoded root certificates as a byte string,
```

Comment:
You can specify that without those params, grpc will use the default CA's [link](https://grpc.io/docs/guides/auth/)